module.info

```diff
@@ -1,5 +1,5 @@
 Name: Cube
-Version: 1.2.2
+Version: 1.3.0
 Requires:
   Libraries: icinga-php-library (>=0.9.0)
```

Comment:
ipl v0.11.0 will be required, as otherwise the control layout is broken. Also, don't forget to update the documentation.